main/windows/frames/viewInstances.ts

```diff
@@ -74,6 +159,15 @@ export default {
 
     // viewInstance.webContents.openDevTools({ mode: 'detach' })
 
+    const testDapps = [
```

Comment:
just confirming we want to keep these in here for now

Reply:
i added these in today before merging to make testing prototype possible, when opening a view it will choose one randomly